.pre-commit-config.yaml

```diff
@@ -14,6 +14,8 @@ repos:
       - id: check-executables-have-shebangs
       - id: check-json
       - id: check-merge-conflict
+        args: ['--assume-in-merge']
+        exclude: "README.rst"
```

Comment:
merge conflict check will give false positive on rst files with 7-character `=======` headings, there is no workaround other than excluding the entire file

https://github.com/pre-commit/pre-commit-hooks/issues/100